.github/workflows/python-test.yaml

```diff
@@ -14,14 +14,23 @@ jobs:
       fail-fast: false
       matrix:
         os: ["ubuntu-20.04"]
-        python_version: ["3.8", "3.9", "3.10", "3.11"]
+        python_version: ["3.9", "3.10", "3.11"]
```

Comment:
Removing 3.8 since we don't currently support it (due to our typing syntax). We can add it back if its an issue for customers.